charts/bitwardenrs/templates/configmap.yaml

```diff
@@ -31,6 +31,19 @@ data:
   {{- end }}
   {{- end }}
   {{- end }}
+  {{- with .Values.bitwardenrs.yubico }}
+  {{- if .enabled }}
+  {{- if .server }}
+  YUBICO_SERVER: {{ .server | quote }}
+  {{- end }}
+  {{- if and (not (and .existingSecret.enabled .existingSecret.clientIdKey)) .clientId }}
+  YUBICO_CLIENT_ID: {{ .clientId | quote }}
+  {{- end }}
+  {{- if and (not .existingSecret.enabled) .secretKey }}
+  YUBICO_SECRET_KEY: {{ .secretKey | quote }}
+  {{- end }}
```

Comment:
the above is fine, but what are your thoughts on all or none? if someone is going to put the key in a secret, chances are they will put the ID there as well.

Reply:
I don't have a strong opinion on that.
Requiring that both variables should be defined in a secret would simplify conditions though, so I'll do that.